### ISSUE TITLE
Bug/labels from scenarios are not passed to alerts

### ISSUE THREAD

**Issue by sabban** [crowdsecurity/crowdsec#1627]
**Describe the bug**
It seems that labels from scenario are not passed into RuntimeAlerts. It's not that important right now as it's not used anywhere, but it would be handy to use it to discriminate into profiles.

**Comment by buixor** [comment]
yes agreed, it would be cool for profiles that would react differently based on that

**Comment by LaurenceJJones** [comment]
Also user has suggested that scenarios can set a value in a label which could be used in the expr for ban duration or ban duration expr.

**Comment by LaurenceJJones** [comment]
Update title and tag to reflect its not a bug and rather a enhance request
